### DISCUSSION TITLE
Closed issues should be purple as in the web version

### DISCUSSION THREAD

**Discussion by nohli** [community/community#7261] [Mobile] [answered]
A solved issue that is shown with a red icon makes me think there is an error.
On github.com, closed issues have a purple icon.

**Comment by mxie** [comment, marked as answer]
Thanks for your feedback @nohli! The purple icons are currently shipped on the beta versions of our app. They will be generally available in the next week or so. Just in case, which OS are you on?

**Reply by nohli** [reply]
Thanks for your answer. Cool!

iOS

**Reply by nohli** [reply]
Purple icons have arrived in the iOS version 😊

**Reply by nohli** [reply]
@mxie the icons for a linked issue within another issue are still red.
Example: https://github.com/flutter/flutter/issues/60118

**Reply by mxie** [reply]
Aha! Thanks for the example - this is super helpful! I'll pass this along to our team.

**Comment by nohli** [comment]
In the latest version, the icon color is consistent between web and iOS app 💯

**Reply by HoneyCivik** [reply]
nohli
